pkg/operation/seed/seed.go

```diff
@@ -693,8 +692,10 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 			return err
 		}
 
+		seedCopy := seed.Info.DeepCopy()
 		seed.Info.Status.ClusterIdentity = &seedClusterIdentity
-		if err := k8sGardenClient.Client().Status().Update(ctx, seed.Info); err != nil {
+
+		if err := k8sGardenClient.Client().Status().Patch(ctx, seed.Info, client.MergeFrom(seedCopy)); err != nil {
```

Comment:
Is it really safe to use a json merge patch here? Imagine there is another controller which updates the status in the meantime, like this controller which maintains the backup bucket condition in the future (#3487). I think this intermediate update will get lost, right?

Reply:
No, it shouldn't get lost, as the patch that the seed controller is sending will only contain the diff (i.e. `.status.clusterIdentity`) and will not overwrite other fields, if they have been updated concurrently.
Please correct me, if I'm wrong.

Reply:
I see, it won't get lost because the conditions aren't modified.